docs/_includes/generated/mesos_configuration.html

```diff
@@ -27,6 +27,11 @@
             <td style="word-wrap: break-word;">-1</td>
             <td>The maximum number of failed workers before the cluster fails. May be set to -1 to disable this feature. This option is ignored unless Flink is in <a href="#legacy">legacy mode</a>.</td>
         </tr>
+        <tr>
+            <td><h5>mesos.maximum-failed-workers-per-interval</h5></td>
+            <td style="word-wrap: break-word;">-1</td>
+            <td>Maximum number of workers the system is going to reallocate in case of a failure in an interval.</td>
```

Comment:
Is this a Mesos specific configuration? If we want to support that for all `ResourceManagers`, then it might be better to use a more generic configuration name. E.g. `resourcemanager.maximum-workers-failure-rate`.

Should we combine `maximum-failed-workers-per-interval` and `workers-failure-rate-interval` into a `failure-rate` config option which defines how many workers can fail per minute or per hour?

Reply:
Rather than separate configs for Yarn and Mesos, I added the failure rate option into resource manager config.